src/modal-create-component.js

```diff
@@ -226,12 +234,18 @@ function addPaths (evt) {
     views.pathAdd.textContent = 'Add root paths:'
     views.pathAdd.addEventListener('click', addPath.bind(this), false);
 
+    views.pathAddAuto = document.createElement('button');
+    views.pathAddAuto.classList.add('inline-block', 'btn', 'btn-warning', 'btn-xs', 'icon', 'icon-file-add');
+    views.pathAddAuto.textContent = 'Add tree view contents'
```

Comment:
I went with the shortest way of clearly expressing the intent of the button, but if you think that `Add tree view root folders` is more precise, that works too. I think `Add root paths` is succinct and communicates what it does well.